README.md

```diff
@@ -31,17 +34,35 @@ func init() {
 	logrus.SetOutput(os.Stderr)
 	logrus.SetLevel(logrus.DebugLevel)
 
+	// Log timestamps in UTC
```

Comment:
Great tip!  Maybe there is something special about "MST"?  I too saw that in other documentation, however at the time I was testing using "UTC" so maybe that wasn't being parsed how I expected?  Explains why I would have to explicitly get the time zone string myself and append it to the pattern since it wasn't being replaced.

✔

Reply:
I had the same confusion around the same time I was writing discordrus. Basically, every token in the format (`Jan`, `2`, `15`, `04`, `05`, `MST`) actually have meanings behind them (month, date of month without trailing zero, hour, minute, seconds, time zone respectively). You'd think you could replace `Jan` with `Feb`, but this wouldn't work since the point of hardcoding the meanings to values is to prevent ambiguity (it wouldn't be very straightforward getting Go to parse `15:15:15000`, for example).

The significance of the values chosen are explained well here:
- https://stackoverflow.com/a/20531191/2102540
- https://stackoverflow.com/a/28087574/2102540